src/commands.py

```diff
@@ -339,3 +339,106 @@ async def action(self, message):
     @staticmethod
     def helptxt():
         return "$help [command] \nDisplays description of the provided command. If none is provided, displays description for all commands."
+
+class machine_dual(command_on_message):
```

Comment:
Let's make a new folder in tools called "examples" and move these 2 commands to a file called "state_machine_examples.py". We wouldn't want these in production.